submissions/permissions.py

```diff
@@ -0,0 +1,11 @@
+"""Permission classes."""
+from rest_framework import permissions
+
+
+class IsXQueueUser(permissions.BasePermission):
+    """
+    Permission classes for submissions app.
+    """
+
+    def has_permission(self, request, view):
+        return request.user.groups.filter(name='xqueue').exists()
```

Comment:
It doesn't have to happen in this PR, but this expectation is going to need to go in the README for edx-submissions and in the release notes.